package.json

```diff
@@ -59,6 +59,7 @@
     "@types/prop-types": "15.7.3",
     "@types/react": "^16.9.33",
     "@types/react-dom": "^16.9.6",
+    "@types/react-transition-group": "^4.4.0",
```

Comment:
If this isn't a global package (scoped to `react-modals`) should we have this `devDependency` in `modals/package.json`?

Reply:
I initially had it under `packages/modals/package.json`, but got a huge explosion of TS errors. I'm going to give it another attempt though.

<img width="818" alt="Screen Shot 2020-09-24 at 12 46 13 PM" src="https://user-images.githubusercontent.com/1811365/94192541-f2fb6300-fe63-11ea-8b69-83f4f8485ad4.png">

Reply:
We may need to introduce an `@types/react` resolution in the root `package.json` if the versions aren't compatible (this probably has to do with the older `@types/styled-components` we're using)

Reply:
>  introduce an @types/react resolution in the root package.json 

No luck so far 😔 .

Reply:
This seems to work:

1) Install `@types/react-transition-group` into `packages/modal`
2) Add `@types/react` to yarn `resolutions`
3) Run `yarn install` which updates the root `yarn.lock` file